client/src/main/java/hudson/plugins/swarm/SwarmClient.java

```diff
@@ -391,6 +398,7 @@ protected void verifyThatUrlIsHudson(Candidate target) throws InterruptedExcepti
         }
     }
 
+    @SuppressWarnings("unused")
```

Comment:
If it's not used we should remove it. Though I would prefer another PR to perform that cleanup. Just to keep things separated.